src/stylesheets/components/_header.scss

```diff
@@ -193,7 +193,7 @@ $z-index-nav:     9000;
       &[aria-expanded=false],
       &[aria-expanded=true] { /* stylelint-disable-line selector-no-qualifying-type */
         @include media($nav-width) { /* stylelint-disable-line selector-no-qualifying-type */
-          background-position: right 1.5rem top 50%;
+          background-position: right 1.5rem top 44%;
```

Comment:
@msecret if you're thinking of this comment https://github.com/18F/web-design-standards/issues/1753#issuecomment-306972426 it wasn't about that it shouldn't be vertically centered, it was about not aligning the hover bar to the baseline. I often use a short header with the nav vertically centered. I've seen some designers align it to the top of the logo tho, different designers may want different treatments, so it may be good to account for this and allow a bit of flexibility.